sd-ran/requirements.yaml

```diff
@@ -0,0 +1,29 @@
+dependencies:
```

Comment:
This chart should also depend on the `atomix-controller` chart, which includes CRDs for Atomix stores.

Reply:
Yes I have added Atomix controller on it. However, the name of Atomix controller is {{ .Release.Name }}. When I deployed the umbrella chart with `helm install sd-ran ...`, the name of Atomix controller was `sd-ran-XXX` which is the {{ .Release.Name }}. Since the readability is poor, I did not add Atomix controller on it. If there is no issue when I change Atomix controller pod's name, I can add Atomix controller in dependencies.

Reply:
Instead of adding the dependency checker or something like that, I just updated document that Atomix controller should be deployed before this chart.